### PR TITLE
Check WebView support and version

### PR DESCRIPTION
**Pull request by Maxr1998**
Closes #300.

### CODE REVIEW

File: app/src/main/java/org/jellyfin/mobile/utils/WebViewUtils.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    // Revert to original UA string
+    settings.userAgentString = originalUA
+
+    return defaultUserAgentString
+}
+
+@SuppressLint("SetJavaScriptEnabled")
+fun WebView.setDefaultSettings() {
+    with(settings) {
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/utils/WebViewUtils.kt:65]:
```suggestion
fun WebView.setDefaultSettings() = with(settings) {
```

File: app/src/main/java/org/jellyfin/mobile/utils/Constants.kt [hunk 2]
```diff
@@ -9,6 +9,7 @@ object Constants {
     const val APP_INFO_VERSION: String = BuildConfig.VERSION_NAME
 
     // Webapp constants
+    const val MINIMUM_WEB_VIEW_VERSION = 88
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/utils/Constants.kt:12]: Any particular reason version 88 was chosen?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,4 +1,11 @@
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+    <string name="dialog_web_view_not_supported">WebView not supported</string>
+    <string name="dialog_web_view_not_supported_message">WebViews are not supported by this device.\nPlease install a compatible WebView implementation if possible or switch to a supported device.</string>
```
Comment (nielsvanvelzen) [inline app/src/main/res/values/strings.xml:3]:
```suggestion
    <string name="dialog_web_view_not_supported_message">WebViews are not supported by this device.\nInstall a compatible WebView implementation or switch to a supported device.</string>
```

No need to beg the user

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/jellyfin/mobile/utils/Constants.kt (hunk 2 above)
Comment (Maxr1998) [reply app/src/main/java/org/jellyfin/mobile/utils/Constants.kt:12]: Tachiyomi uses the same, so it seemed sane.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Maxr1998) [reply app/src/main/res/values/strings.xml:3]: lol, kk

**Review by nielsvanvelzen — APPROVED** [review]

**Review by nielsvanvelzen — DISMISSED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**